examples/correlation_alignment_example.py

```diff
@@ -75,6 +75,8 @@
         fl.image(data)
         data.ft("t2")
         fl.next("After hermitian function test -- Frequency domain")
+        # both of these plots labeled "After hermitian function test" now show
+        # the sign of the signal bouncing back and forth -- why is that!!!????
```

Comment:
address/fix this, then remove comment

Reply:
Okay so after some debugging I find that this is a result of extending the echo time. When the echo time is 1e-3 it works as it should... but when we increase this it falls apart... I believe this has to do with the hermitian function as it stands right now but I am unsure. I changed the echo time back to 1ms as this seems to work. I don't know how I would go about fixing it when the echo time is 10 ms as I think that would be editing the hermitian function test which should be another PR separate from this.

Reply:
OK, then you need a separate PR to demo the hermitian function test.  My response here would be to look and see what your cost function looks like.

Reply:
![image](https://user-images.githubusercontent.com/52217688/124659664-33682280-de73-11eb-9350-d8b3b47af695.png)
My cost function looks as it should and the hermitian does center the echo at 0 so I'm not entirely sure what the issue is?

Reply:
yes, so make a new branch where you make a copy of this code as a new example, and just delete everything after the part where you apply the hermitian phase correction.